### PR TITLE
a typo fix in the git/desktop/homework file

### PR DESCRIPTION
**Pull request by hussein-alsayed**
I noticed the word 'have' might be missing, so I thought I'd put what I have learnt into practical practice, find this file on github, and suggest the fix.

Signed-off-by: Hussein Al-Sayed <111397062+hussein-alsayed@users.noreply.github.com>

## What does this change?

Module: Git and Github
Week(s): 1

## Description

I added what I think might be a missing word in line 25 "When you've completed this step you should -have- a.."
'have' was missing.

## Who needs to know about this?

@shieldo @SallyMcGrath @ClaireBickley

### CODE REVIEW

**Review by gregdyke — APPROVED** [review]
Yes! We love it when trainees help fixing the syllabus. Nice one!

**Comment by gregdyke** [comment]
I'll let you do the honours and press "merge pull request"

**Comment by hussein-alsayed** [comment]
Thank you, @gregdyke 😂 I can now say I'm a CYF syllabus contributor!

**Comment by hussein-alsayed** [comment]
it says only those with write access can merge it :')

I felt the honour tho 👍

**Comment by shieldo** [comment]
@hussein-alsayed excellent - thanks for this correction!

I'll merge it and we can pretend that you did :)

**Pull request merged**
